code/modules/events/holiday/halloween.dm

```diff
@@ -40,6 +40,9 @@
 	icon_state = "coffincookie"
 	crafting_complexity = FOOD_COMPLEXITY_2
 
+/obj/item/food/cookie/sugar/spookycoffin/Initialize(mapload, seasonal_changes = FALSE)
+	return ..()
```

Comment:
If you're doing something that looks a bit obtuse / needless at a glance, you should leave a comment so someone doesn't clean it up by accident
```suggestion
/obj/item/food/cookie/sugar/spookycoffin/Initialize(mapload, seasonal_changes = FALSE)
	// Changes default parameter of seasonal_changes to FALSE, pass to parent
	return ..()
```